p2p/simulations/network.go

```diff
@@ -642,7 +710,7 @@ func (self *Network) DidConnect(one, other *adapters.NodeId) error {
 	conn.Reverse = conn.One.NodeID != one.NodeID
 	conn.Up = true
 	// connection event posted
-	self.events.Post(conn.event(true, conn.Reverse))
+	self.events.Post(conn.EmitEvent(LiveEvent))
```

Comment:
On lines 703 and 718
we need 
```
conn, err := self.GetOrCreateConn(oneId, otherId)
```

otherwise live events (naturally occuring conn/disconn via hive) are not gonna be found